tools/FakeItEasy.Build/Program.cs

```diff
@@ -56,6 +56,19 @@
     forEach: projectsToPack,
     action: project => Run("dotnet", $"pack {project.Path} --configuration Release --no-build --nologo --output \"{Path.GetFullPath("artifacts/output")}\""));
 
+// If mkdocs's site_url configuration is not set, sitemap.xml is full of "None" links.
+// Even with a valid site_url, the link check would fail if we added a new page.
+// So trust that sitemap.xml will be generated correctly.
+// The 404.html page is likewise generated, and isn't even the one we use on the live site (mike generates a new one).
+Target(
+    "docs",
+    DependsOn("generate-docs"),
+    () => Run("linkchecker", "--ignore-url=sitemap.xml --ignore-url=404.html --check-extern -F html/./artifacts/docs-link-check.html ./artifacts/docs/index.html"));
+
+Target(
+    "generate-docs",
+    () => Run("python", "-m mkdocs build --clean --site-dir artifacts/docs --config-file mkdocs.yml --strict"));
+

```

Comment:
I waffled over how to structure the targets.
Coulda been one big target: build and check.
Coulda been 3 targets: docs that depends on check-docs-links that depends on generate-docs
Coulda been 3 targets: docs that depends on both generate-docs and check-docs-links (with checks-docs-links depending on nothing)

I went this way but would go another way!

Reply:
I think I prefer this one:

> 3 targets: docs that depends on check-docs-links that depends on generate-docs

It's a bit strange that the `docs` target actually does the check